analyzer/src/main/kotlin/managers/Poetry.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
```

Comment:
@mnonnenmacher how should I treat the copyright? update the year? do you have any examples to look at?

Reply:
The year must be 2022, and you are not making this contribution in the name of HERE Europe B.V., so if this is a private contribution the statement should be your name, otherwise the name of the company you are doing this contribution for.